viamonvif/service.go

```diff
@@ -82,6 +83,10 @@ type rtspDiscovery struct {
 	Credentials []device.Credentials
 	mdnsServer  *mdnsServer
 	logger      logging.Logger
+
+	workers *utils.StoppableWorkers
+
+	discoveredResources []resource.Config
```

Comment:
Need to add mutex since discoveredResources can also be modified by DiscoverResources API